pennylane/templates/subroutines/qsvt.py

```diff
@@ -1003,11 +1240,16 @@ def circuit_qsvt():
     if routine == "QSVT":
         if angle_solver == "root-finding":
             return transform_angles(_compute_qsp_angle(poly), "QSP", "QSVT")
+        if angle_solver == "iterative":
+            return transform_angles(_compute_qsp_angles_iteratively(poly), "QSP", "QSVT")
+
         raise AssertionError("Invalid angle solver method. We currently support 'root-finding'")
 
     if routine == "QSP":
         if angle_solver == "root-finding":
             return _compute_qsp_angle(poly)
+        if angle_solver == "iterative":
+            return _compute_qsp_angles_iteratively(poly)
         raise AssertionError("Invalid angle solver method. Valid value is 'root-finding'")
 
     if routine == "GQSP":
```

Comment:
Mind also help correct the next assertion error message below? I can't make comment on webui of gh